x-pack/docs/en/security/ccs-clients-integrations/http.asciidoc

```diff
@@ -46,7 +50,22 @@ curl --user rdeniro:taxidriver -XPUT 'localhost:9200/idx'
 }
 ---------------------------------------------------------
 
-[float]
+[discrete]
+[[http-clients-secondary-authorization]]
+==== Secondary authorization
+
+Some APIs support secondary authorization headers for situations where you want
+tasks to run with a different set of credentials. For example, you can send the
+following header in addition to the basic authentication header:
+
+[source,shell]
+--------------------------------------------------
+Authorization: es-secondary-authorization <TOKEN> <1>
```

Comment:
This should be 
```suggestion
es-secondary-authorization: basic <TOKEN> <1>
```

Instead of "basic" we can support "bearer" or "apikey" there in the same was we support that for `Authorization` headers.
Basically, `es-secondary-authorization` has the same syntax as the`Authorization` header.